dm/checker/checker.go

```diff
@@ -119,75 +122,51 @@ func (c *Checker) Init(ctx context.Context) (err error) {
 	rollbackHolder.Add(fr.FuncRollback{Name: "close-DBs", Fn: c.closeDBs})
 
 	c.tctx = tcontext.NewContext(ctx, log.With(zap.String("unit", "task check")))
+
+	// prepare source target do dbs concurrently
+	eg, ctx2 := errgroup.WithContext(ctx)
+	// sourceID => source targetDB mapping, only writing need used mu to avoid concurrent write map
+	var mu sync.Mutex
+	sourceTargetM := make(map[string]map[string][]*filter.Table)
```

Comment:
nit: if you use preallocated slice to store `map[string][]*filter.Table` and let i-th c.instances access i-th element, there should be no data race